### ISSUE TITLE
Run accessibility reports for Phase 3 - April Lockler

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#709]

**Comment by Maroyafaied** [comment]
https://doimspp-my.sharepoint.com/:x:/r/personal/mchargs_mms_gov/_layouts/15/Doc.aspx?sourcedoc=%7B928732E3-BA62-4E42-8D4D-A815C2802C66%7D&file=AccessibilityTracking.xlsx&action=default&mobileredirect=true

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 Below are instructions for this issue, please let me know if you have any questions or feel free to schedule a meeting if you want me to do one as an example. 
1. Filter the [accessibility tracking tab](https://doimspp-my.sharepoint.com/:x:/r/personal/mchargs_mms_gov/_layouts/15/Doc.aspx?sourcedoc=%7B928732E3-BA62-4E42-8D4D-A815C2802C66%7D&file=AccessibilityTracking.xlsx&action=default&mobileredirect=true) for April Lockler (column L), not accessible (column Q), phase 3 (column R)
2.  Run accessibility check  and report for each file (column E). Instruction can be found [here ](https://github.com/ONRR/onrr.gov-site/wiki/Checking-the-accessibility-of-a-PDF)
3.  Save accessibility report in this [folder](https://doimspp-my.sharepoint.com/personal/mchargs_mms_gov/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fmchargs%5Fmms%5Fgov%2FDocuments%2Fonrr%2Egov%2F508%2FPhase%203%20Accessibility%20Reports) and add link to the report in column O for each file

**Comment by mcharg** [comment]
@daniel-coleman-1 Column Q is actually going to be blank for the ones that don't have a report.  Once you run the report, you can update that column with either Accessible or Not Accessible.

**Comment by mcharg** [comment]
And add your name to column P as the one who generated the report.

**Comment by daniel-coleman-1** [comment]
> @daniel-coleman-1 Column Q is actually going to be blank for the ones that don't have a report. Once you run the report, you can update that column with either Accessible or Not Accessible.

I'm not seeing any that don't have a report - just Accessible, Not Accessible, or Removed.  Am I filtering them incorrectly?

**Comment by Maroyafaied** [comment]
@daniel-coleman-1  This is what you should be filtering for: April Lockler (column L), Blank (column Q), phase 3 (column R).  Let me know if that makes sense.
![image](https://user-images.githubusercontent.com/40175688/118734060-d056f980-b7fa-11eb-8aee-a7582bc74e2e.png)

**Comment by mcharg** [comment]
@daniel-coleman-1 I put your name in column P for the ones that need checking, in case it's easier to filter by that.
